src/graph/graph.c

```diff
@@ -280,11 +280,13 @@ MATRIX_POLICY Graph_GetMatrixPolicy
 }
 
 // define the current behavior for matrix creations and retrievals on this graph
-void Graph_SetMatrixPolicy
+MATRIX_POLICY Graph_SetMatrixPolicy
 (
 	Graph *g,
 	MATRIX_POLICY policy
 ) {
+	MATRIX_POLICY prev_policy = Graph_GetMatrixPolicy(g);
```

Comment:
in the first call probably this is not initialize

Reply:
When creating a new Graph via `Graph_New` the policy is set.